src/interface_r/R/package.R

```diff
@@ -0,0 +1,43 @@
+#' h2o4gpu in R
+#' 
+#' @docType package
+#' @name h2o4gpu
+NULL
+
+h2o4gpu <- NULL
+np <- NULL
+
+.onLoad <- function(libname, pkgname) {
+  
+  # delay load handler
+  displayed_warning <- FALSE
+  delay_load <- list(
+    
+    priority = 5,
+    
+    environment = "r-h2o4gpu",
+    
+    on_load = function() {
+      check_compatibility(displayed_warning)
+      search_env <- "r-h2o4gpu"
+      conda_path <- "/home/terry/miniconda3/bin/conda"
+      if (search_env %in% reticulate::conda_list(conda = conda_path)$name) {
+        use_condaenv(search_env, conda = conda_path)
+      } else {
+        message("h2o4gpu Python package is not installed. Please use install_h2o4gpu(path_to_wheel) to install.")
```

Comment:
There is a s3 path to a .whl file. Shown in the h2o4gpu README

Reply:
Yes, this is a method that user can call and they can specify a path, include s3 path. I'll add more docs in future PRs.